examples/eth-with-802-1x.yml

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: network-test
+  vars:
+    network_connections:
+      - name: eth0
+        type: ethernet
+        802.1x:
+          identity: myhost
+          private-key: /etc/pki/tls/client.key
+          # recommend vault encrypting the private key password
```

Comment:
Please add a link to the vault docs here.

Reply:
added link